pkg/process/config/yaml_config.go

```diff
@@ -56,12 +56,11 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 		a.HostName = config.Datadog.GetString("hostname")
 	}
 
-	a.Enabled = false
 	if config.Datadog.GetBool("process_config.process_collection.enabled") {
-		a.Enabled, a.EnabledChecks = true, processChecks
+		a.EnabledChecks = processChecks
```

Comment:
Should we add the processes and containers checks here by concatenating the two slices?
Something like
```
a.EnabledChecks = append(a.EnabledChecks, processChecks...)
```

This might prevent some errors in the future if we add other checks before these two.

Reply:
That makes a lot of sense. I changed these to append